### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by PavlovaOlena**
[DEMO LINK](https://PavlovaOlena.github.io/layout_calendar/)
[TEST REPORT LINK](https:///PavlovaOlena.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__month-length--28">
+      <div class="calendar__field">
+        <div class="calendar__cell calendar__cell-start-day--sun"></div>
```
Comment (witflash) [inline src/index.html]: You should use "start-day" modifier to the whole "calendar" block

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+  }
+
+  @for $index from 1 through 31 {
+    &__cell:nth-child(#{$index})::before {
+      content: "#{$index}";
+    }
+  }
+
+  &__month-length {
```
Comment (witflash) [inline src/styles/blocks/calendar.scss]: Use this modifier for the "calendar" selector

**Review by witflash — CHANGES_REQUESTED** [review]
Good job 👍🏻 
Just move modifier to the block instead of single element

File: src/index.html (hunk 1 above)
Comment (PavlovaOlena) [reply src/index.html]: Done

File: src/styles/blocks/calendar.scss (hunk 2 above)
Comment (PavlovaOlena) [reply src/styles/blocks/calendar.scss]: Done

**Review by qoa11a — APPROVED** [review]

**Pull request closed without merging**